morphocut/stream.py

```diff
@@ -15,9 +15,18 @@
 
 class TQDM(Node):
     """
-    Provide a progress indicator via `tqdm`_.
+    Provide a progress indicator via `tqdm`_. Returns an iterator which prints a dynamically
+    updating progressbar.
```

Comment:
https://trello.com/c/ub9i0YrA/14-write-a-docstring

Summary is ok. But this returns nothing.